src/types/node.d.ts

```diff
@@ -183,52 +197,69 @@ export type NodeOffice = DrupalNode & {
   field_social_media_links: FieldSocialMediaLinks
 }
 
-export type NodePersonProfile = DrupalNode & {
+/** A representation of a staff member of the VA or a VA facility.
+ *
+ *  @see https://prod.cms.va.gov/admin/structure/types/manage/person_profile/fields
+ */
+export interface NodePersonProfile extends DrupalNode {
+  /** The bio text displayed on the page. */
   field_body: string
+  /** A PDF containing an official biography for distribution. */
   field_complete_biography: DrupalFile
+  /** Whether to create a biography. */
   field_complete_biography_create: boolean
+  /** The email address of the person. */
   field_email_address: string
+  /** First name. */
   field_name_first: string
+  /** Introductory text. */
   field_intro_text: string
+  /** Whether this person's photo is allowed to be downloaded. */
   field_photo_allow_hires_download: boolean
+  /** A brief description of this person's role. */
   field_description: string
+  /** Last name. */
   field_last_name: string
+  /** Phone number. */
   field_phone_number: string
+  /** A photo of the person. */
   field_media: MediaImage
+  /** The office or facility which this person is associated with. */
   field_office: NodeOffice | NodeHealthCareRegionPage
+  /** Any honorific suffix, i.e. MD, LCSW, PhD, etc. */
   field_suffix: string
 }
 
-export type NodePromoBanner = DrupalNode & {
+export interface NodePromoBanner extends DrupalNode {
   field_target_paths: string[]
   field_promo_type: string
   field_link: FieldLink
 }
 
-export type NodeQA = NodeAbstractResource & {
+export interface NodeQA extends NodeAbstractResource {
   field_answer: ParagraphRichTextCharLimit1000
   field_standalone_page: boolean
   field_q_a_groups: ParagraphQAGroup[]
 }
 
-export type NodeRegionalHealthCareServiceDes = NodeAbstractResource & {
+export interface NodeRegionalHealthCareServiceDes extends NodeAbstractResource {
   field_local_health_care_service_: NodeHealthCareLocalHealthService[]
   field_service_name_and_descripti: TaxonomyTermHealthCareServiceTaxonomy[] //@todo
   field_region_page: NodeHealthCareRegionPage
   field_body: string
 }
-export type NodeStepByStep = NodeAbstractResource & {
+export interface NodeStepByStep extends NodeAbstractResource {
```

Comment:
is `NodeAbstractResource` the correct type for Step by Step? I'm not very clear on the difference between `DrupalNodes` & `NodeAbstractResource`